web/app/view/MapMarkerController.js

```diff
@@ -346,11 +347,13 @@ Ext.define('Traccar.view.MapMarkerController', {
     selectMarker: function (marker, center) {
         if (this.selectedMarker) {
             this.resizeMarker(this.selectedMarker.getStyle(), false);
+            this.selectedMarker.getStyle().setZIndex(0);
             this.selectedMarker.changed();
         }
 
         if (marker) {
             this.resizeMarker(marker.getStyle(), true);
+            marker.getStyle().setZIndex(1);
```

Comment:
I cant find the documentation explaining this, but if I use -1 marker become lower than others.